snuba/snuba_migrations/events_analytics_platform/0001_spans.py

```diff
@@ -158,10 +158,12 @@ def backwards_ops(self) -> Sequence[SqlOperation]:
                 storage_set=storage_set_name,
                 table_name=local_table_name,
                 target=OperationTarget.LOCAL,
+                sync=True,
```

Comment:
What do we plan to do after changing this migration in-line?

Reply:
should probably wait for @onewland - we'll have to drop all of the data again because we are changing the storage configuration of the clickhouse cluster